pkgs/development/python-modules/hwi/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, mnemonic
+, ecdsa
+, typing-extensions
+, hidapi
+, libusb1
+, pyaes
+, trezor
+, btchip
+, ckcc-protocol
+}:
+
+buildPythonPackage rec {
+  pname = "hwi";
+  version = "1.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "eec460a51eb556500c1eca92015be246d5714cd53171407a76da71e4346048ae";
+  };
+
+  propagatedBuildInputs = [ mnemonic ecdsa typing-extensions hidapi libusb1 pyaes trezor btchip ckcc-protocol ];
```

with so many inputs, do you mind listing one per line
```suggestion
  propagatedBuildInputs = [
    mnemonic
    ecdsa
    ...
  ];
```

---

Fixed in 12d262130bd4153ad77930b243f9257f37c2795f